mm2src/mm2_main/src/lp_healthcheck.rs

```diff
@@ -121,11 +121,43 @@ impl HealthcheckMessage {
         Ok(Self { signature, data })
     }
 
-    pub(crate) fn is_received_message_valid(
-        &self,
-        my_peer_address: PeerAddress,
-        healthcheck_config: &HealthcheckConfig,
-    ) -> bool {
+    fn generate_or_use_cached_message(ctx: &MmArc) -> Result<Self, String> {
+        const MIN_DURATION_FOR_REUSABLE_MSG: Duration = Duration::from_secs(5);
```

Comment:
nitty nit: idk much about the propagation perf of our p2p network, but i would set this to be like one minute to be on the safe side. so also the 10s expiration to 2 minutes maybe.

Reply:
Message generation once per 5 seconds is already more than enough. Increasing this duration will lead false-possitives on the security side.